docs/src/protocol_library.md

```diff
@@ -25,41 +25,48 @@ Most procedures in the Miden protocol library are implemented as wrappers around
 
-## Account Procedures (`miden::account`)
-
-Account procedures can be used to read and write to account storage, add or remove assets from the vault and fetch or compute commitments.
-
-| Procedure                        | Description                                                                                                                                                                                               | Context          |
-| -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
-| `get_id`                         | Returns the ID of the current account.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[account_id_prefix, account_id_suffix]`                                                                                | Any              |
-| `get_native_id`                  | Returns the ID of the native account of the transaction.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[account_id_prefix, account_id_suffix]`                                                              | Any              |
-| `get_nonce`                      | Returns the nonce of the current account. Always returns the initial nonce as it can only be incremented in auth procedures.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[nonce]`                         | Any              |
-| `get_native_nonce`               | Returns the nonce of the native account of the transaction.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[nonce]`                                                                                          | Any              |
-| `incr_nonce`                     | Increments the account nonce by one and returns the new nonce. Can only be called from auth procedures.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[final_nonce]`                                        | Auth             |
-| `get_initial_commitment`         | Returns the native account commitment at the beginning of the transaction.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[INIT_COMMITMENT]`                                                                 | Any              |
-| `compute_current_commitment`     | Computes and returns the account commitment from account data stored in memory.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[ACCOUNT_COMMITMENT]`                                                         | Any              |
-| `compute_delta_commitment`       | Computes the commitment to the native account's delta. Can only be called from auth procedures.<br/><br/>**Inputs:** `[]`<br/>**Outputs:** `[DELTA_COMMITMENT]`                                           | Auth             |
+## Active account Procedures (`miden::active_account`)
```

Comment:
Nit: Should we put these into the same order as they appear in `active_account.masm`?